app/models/user.rb

```diff
@@ -97,10 +97,11 @@ def self.find_by_credentials(login, password)
 
   def self.search_qualifiers
     [
-      Search::Qualifier.new(:name => "name", :col => "users.name", :type => :text, :default => true),
-      Search::Qualifier.new(:name => "login", :col => "users.login", :type => :text, :default => true),
-      Search::Qualifier.new(:name => "email", :col => "users.email", :type => :text, :default => true),
-      Search::Qualifier.new(:name => "phone", :col => "users.phone", :type => :text)
+      Search::Qualifier.new(name: "name", col: "users.name", type: :text, default: true),
+      Search::Qualifier.new(name: "login", col: "users.login", type: :text, default: true),
+      Search::Qualifier.new(name: "email", col: "users.email", type: :text, default: true),
+      Search::Qualifier.new(name: "phone", col: "users.phone", type: :text),
+      Search::Qualifier.new(name: "group", col: "user_groups.name", type: :text, assoc: :user_groups)
```

Comment:
Does this really work? Might be good to add a quick spec for this modeled off of question_search_spec?